docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -135,7 +135,13 @@ property key | Description | Default |
 `correlation.element_max_number`|Max element count of the correlation context.|`3`|
 `correlation.value_max_length`|Max value length of correlation context element.|`128`|
 `plugin.kafka.bootstrap_servers`| A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | `localhost:9092`
-`plugin.kafka.consumer_config`| Kafka producer configuration. |
+`plugin.kafka.consumer_config`| Kafka producer configuration. ||
+`plugin.dubbo.collect_consumer_address`| Apache Dubbo consumer collect `remote_address` and `local_address` in RPC call. |`false`| 
```

Comment:
Isn't `remote == peer`? I am a little confused.

Reply:
`remote_address` and `local_address` is the port and IP of  Netty channel

Reply:
What does today's peer look like?

Reply:
> Isn't `remote == peer`? I am a little confused.

yes, when dubbo consumer,  remote == peer

Reply:
Why doesn't use peer in the entry span to collect this remote-IP directly? Also, at the same time, isn't local IP should be a part of instance information? Do you have concerns about the multiple IPs issue?

Reply:
`plugin.dubbo.collect_consumer_address ` is redundant. I deleted it